src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -626,6 +634,17 @@ private OpenApiMediaType CreateResponseMediaType(ModelMetadata modelMetadata, Sc
             };
         }
 
+        private static bool IsFromFormAttributeUsedWithIFormFile(ApiParameterDescription apiParameter)
+        {
+            // Retrieve parameter information
+            var parameterInfo = apiParameter.ParameterInfo();
+
+            // Check if Parameter has FromForm Attribute
+            var fromFormAttribute = parameterInfo?.GetCustomAttribute<FromFormAttribute>();
+
+            return fromFormAttribute != null && parameterInfo?.ParameterType == typeof(IFormFile);
```

Comment:
The code here is pretty self-explanatory, so I don't think the comments add anything:

```suggestion
        private static bool IsFromFormAttributeUsedWithIFormFile(ApiParameterDescription apiParameter)
        {
            var parameterInfo = apiParameter.ParameterInfo();
            var fromFormAttribute = parameterInfo?.GetCustomAttribute<FromFormAttribute>();

            return fromFormAttribute != null && parameterInfo?.ParameterType == typeof(IFormFile);
```

Reply:
ok. Done